### ISSUE TITLE
Enhance Data Browser with Tabs

### ISSUE THREAD

**Issue by horst-p-w-neubauer** [sqlitebrowser/sqlitebrowser#2229]
Enhance the Data Browser with tabs that allows you to browse parallel in different (or even the same) tables.
perhaps like this:
![grafik](https://user-images.githubusercontent.com/28151942/80869182-282dcb80-8c9f-11ea-96ca-2628b4a9117f.png)

**Comment by justinclift** [comment]
Yep, this would be useful for lots of people. :smile:

**Comment by mgrojo** [comment]
Yes, this was mentioned before and there are probably some issue already. We have a problem with so many issues, it's difficult to find if something has been asked for before.

@MKleusberg prepared the code to make this enhancement easier some time ago. It could happen for the 3.13 release.

Another option to what you have proposed, would be to allow multiple instances of the Browse Data main tab. I think Martin mentioned another third option, more powerful but requiring more work. Everything should be a dock, so you could arrange the window and see two Data Browsers side-by-side, and either attached or detached from the main window.

**Comment by justinclift** [comment]
Ahhh, multiple docks side by side would be optimal. People could figure out their own screen layout as suited for the given task.

**Comment by horst-p-w-neubauer** [comment]
I can't find other open issues on browse data than #1905 which is not exactly the same. 
@mgrojo  or @MKleusberg - if you know of other open issues on this topic or you're allready working on this stuff, I'd propose to join all of them together in one, assign it and close the others as duplicates. That would help to keep open issues small.
@justinclift,  Maybe it could be helpfull to have a label for every main window tab to filter issues.

**Comment by horst-p-w-neubauer** [comment]
Time to market matters. 
Enhancing the code to support multipel Data Browser panels - regardless comming as tab or as docks - is the prior feature that I'm looking for. Changing vom Tabs to docks is nice to have and will be a next step. But if this change slows down the read enhancement - I'd vote for seperating both of them and go for the multipel Data Browser panels first.

**Comment by mgrojo** [comment]
@horst-p-w-neubauer, this is the Martin's change I talked about: 3f0832f449f0bed02e245661408c71d5f12f5c75 From there you can get to three issues mentioning a redesign of the single Data Browse tab and proposing interesting alternatives. I don't mind if you want to close some of them as duplicates, provided that they are properly linked and their subscribers warned. We don't usually do that, but maybe it's time to rethink our strategy, so we aren't overwhelmed with so many issues. In the future, we could also proactively open our own issues with the enhancements we plan to do at some time or would like to have, so we can redirect people to them.

The same goes for labelling, I've shyly added and used some to better classify issues, but I was unsure about how to partition the topics and haven't given much thought about it. This is worth also of a debate, but for the moment, our top priority should be to move 3.12 forward.

**Comment by MKleusberg** [comment]
Yep, I'm definitely planning to add this for 3.13. It's one of the top items on my list of things to do after the 3.12 release :smile:

**Comment by MKleusberg** [comment]
I have just added support for multiple tabs (albeit not docks). Do you want to give it a try, @horst-p-w-neubauer? :smile:
